src/mobot_client/payments/client.py

```diff
@@ -98,6 +98,7 @@ def get_receipt_status(self, receipt: str) -> dict:
             receipt_status = self.check_receiver_receipt_status(
```

Comment:
I love that this commit was separated out the way it was; super easy to understand intent, especially considering how new I am to the codebase 😁